crates/ruff_linter/src/rules/ruff/rules/asyncio_dangling_task.rs

```diff
@@ -80,19 +82,29 @@ pub(crate) fn asyncio_dangling_task(expr: &Expr, semantic: &SemanticModel) -> Op
             })
     {
         return Some(Diagnostic::new(
-            AsyncioDanglingTask { method },
+            AsyncioDanglingTask {
+                expr: "asyncio".to_string(),
+                method,
+            },
             expr.range(),
         ));
     }
 
-    // Ex) `loop = asyncio.get_running_loop(); loop.create_task(...)`
+    // Ex) `loop = ...; loop.create_task(...)`
     if let Expr::Attribute(ast::ExprAttribute { attr, value, .. }) = func.as_ref() {
         if attr == "create_task" {
             if typing::resolve_assignment(value, semantic).is_some_and(|call_path| {
-                matches!(call_path.as_slice(), ["asyncio", "get_running_loop"])
+                matches!(
+                    call_path.as_slice(),
+                    [
+                        "asyncio",
+                        "get_event_loop" | "get_running_loop" | "new_event_loop"
+                    ]
+                )
             }) {
                 return Some(Diagnostic::new(
                     AsyncioDanglingTask {
+                        expr: compose_call_path(value).unwrap_or_else(|| "asyncio".to_string()),
```

Comment:
I think `loop` would be a better fallback:
```suggestion
                        expr: compose_call_path(value).unwrap_or_else(|| "loop".to_string()),
```

Reply:
Will change, though I think in practice this should never happen. I wish it were encoded in the type system, but e.g. `value` here has to be a `Name`, and `compose_call_path` always returns `Some` given a name.